### ISSUE TITLE
[libs/movex-server] Add nx runner for `tsc` compiler 

### ISSUE THREAD

**Issue by GabrielCTroia** [movesthatmatter/movex#100]
# Where

`libs/movex-server/project.json`

```json
{
  "name": "movex-server",
  "$schema": "../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/movex-server/src",
  "projectType": "library",
  "targets": {
    "build": {
      "executor": "@nrwl/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
      ...
      }
    },
    ...
  },
}

```

# What

Add the following piece of code, **in the `targets` object just above the  `build` property:**

```json
"tsc": {
      "executor": "nx:run-commands",
      "options": {
          "commands": [
              {
                  "command": "tsc --noEmit -p libs/movex-server/tsconfig.lib.json"
              }
          ]
      }
    }
```

# Why

To be able to run `npx nx tsc movex-server` or `npx nx tsc movex-server --watch` and add the typescript compiler command to CI.

This will be added as a Gtihub action so every new PR will also be checked for typescript compilation errors.

# Definition of done
- [ ] can run `npx nx tsc movex-server`
- [ ] can run `npx nx tsc movex-server --watch`

**Comment by Abhishek-Mallick** [comment]
@GabrielCTroia Please assign this to me

**Comment by techemmy** [comment]
@Abhishek-Mallick I've raised a PR for this already. @GabrielCTroia can I be assigned this task?

**Comment by GabrielCTroia** [comment]
Sure, @techemmy this is your since you've already started the PR. 

@Abhishek-Mallick thank you for you effort but it seems @techemmy was first! Were you able to find other issues? 🙏

**Comment by GabrielCTroia** [comment]
This was closed in #114
